model-optimizer/mo/ops/slice.py

```diff
@@ -13,133 +13,181 @@
  See the License for the specific language governing permissions and
  limitations under the License.
 """
-
-import logging as log
+from typing import List
 
 import numpy as np
 
 from mo.graph.graph import Node, Graph
 from mo.ops.op import Op
+from mo.utils.error import Error
+
+"""
+Slicing operations have different semantic or different parameters/inputs in different frameworks. To distinguish them 
+and not confuse with OpenVINO Slice several Model Optimizer internal operations are introduced. OpenVINO Slice operation 
```

Comment:
there is no OpenVINO Slice operation. We should not confuse people

Reply:
Agree. Clarified this. Please review.